### ISSUE TITLE
Add completion for declared psalm templates

### ISSUE THREAD

**Issue by isfedorov** [JetBrains/phpstorm-psalm-plugin#25]
Create code
```
<?php
/**
 * @template MyTemp
 */
class MyContainer {
    /** @var MyT<caret> */
    private $value;
}
```
Invoke completion
Expected result:
Completion should suggest `MyTemp` template as possible type
Issue is generally subtask of #16

**Comment by wbars** [comment]
Adding to not only templates, but `@psalm-type` and imported types as well

**Comment by wbars** [comment]
fixed by 03cb7b33cd48e31cf8551e91f0af06b7c72ed730
